src/datashare/azext_datashare/generated/_params.py

```diff
@@ -0,0 +1,307 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azext_datashare.action import AddIdentity
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('datashare account list') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare account show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+
+    with self.argument_context('datashare account create') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location of the azure resource.')
+        c.argument('tags', tags_type, help='Tags on the azure resource.')
+        c.argument('identity', action=AddIdentity, nargs='+', help='Identity of resource')

```

Comment:
from sample, value of identity could be keyvalue pair like type=systemassigned. how about improve the help message

Reply:
Used long-summary in _help.py:
https://github.com/Azure/azure-cli-extensions/blob/217ef67544982e1d40e49d3650119348530b8ff5/src/datashare/azext_datashare/manual/_help.py#L42-L46

Reply:
Long summary is not shown in Azure CLI reference. I have reported it to doc team. But they do not give a satisfactory reply. Be careful using long summary.

Reply:
I will add examples for both type then.

Reply:
I tested the command and only systemAssigned identity is supported now. I think we can hide this option for now and always pass type= systemAssigned  to service.